src/main/java/com/spotify/confidence/Confidence.java

```diff
@@ -70,8 +76,13 @@ public void send(String name, ConfidenceValue.Struct message) {
     eventSenderEngine.send(name, message, getContext());
   }
 
+  ResolveFlagsResponse resolveFlags(String flagName) {
```

Comment:
This API (return type + params) would change if/when we open it up for public use.

Reply:
I believe this is a good start.